src/test/groovy/com/stormpath/tck/me/MeIT.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.jayway.restassured.http.ContentType
+import com.jayway.restassured.response.Cookie
+import com.jayway.restassured.response.Cookies
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.*
+import com.stormpath.tck.responseSpecs.*
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.*
+import static org.testng.Assert.*
+import static org.hamcrest.Matchers.*
+
+@Test
+class MeIT extends AbstractIT {
+    private TestAccount account = new TestAccount()
+    private String accessToken
+
+    @BeforeClass
+    private void getTestAccountAccessToken() throws Exception {
+        accessToken =
+            given()
+                .param("grant_type", "password")
+                .param("username", account.username)
+                .param("password", account.password)
+            .when()
+                .post(FrameworkConstants.OauthRoute)
+            .then()
+                .statusCode(200)
+                .contentType(ContentType.JSON)
+                .body("access_token", not(isEmptyOrNullString()))
+            .extract()
+                .path("access_token")
+        deleteOnClassTeardown(account.href)
+    }
+
+    @Test
```

Comment:
For instance, I believe that these issues are excessive: 

<img width="578" alt="screen shot 2016-03-30 at 6 38 34 pm" src="https://cloud.githubusercontent.com/assets/918770/14163082/ad752466-f6a6-11e5-8091-6b34ecd554e7.png">

We should try to separate concerns into orthogonal tests, so that we have less test code to maintain. These 8 tests could be these five instead:

* Test that registration errors on a server error
* Test that registration errors when there's an undefined custom field
* Test that registration errors if a required custom field is missing
* Test that registration errors if a required field is missing
* Test that registration HTML re-renders the form with an error on an error